build/projects/Extensions (W1)/.AL-Go/settings.json

```diff
@@ -0,0 +1,27 @@
+{
+    "projectName": "Extensions (W1)",
+    "appFolders":  [
+        "../../../src/Extensions/W1/*/app"
```

Comment:
"Add-ons" instead of "Extensions"?

I remember there was a discussion about the naming before.